passkey-types/src/webauthn/attestation.rs

```diff
@@ -569,13 +791,78 @@ pub enum ClientDataType {
     /// Serializes to the string `"webauthn.get"`
     #[serde(rename = "webauthn.get")]
     Get,
+
+    /// Serializes to the string `"payment.get"`
+    #[serde(rename = "payment.get")]
+    PaymentGet,
+
+    /// Serialize to an empty string `""`
+    #[serde(rename = "")]
+    Empty,
```

Comment:
This would not be necessary anymore, and I'm not a fan of having something that isn't actually within spec that could be representable.

Reply:
See above